.github/workflows/build_scoped_rector_php70.yaml

```diff
@@ -34,17 +33,22 @@ jobs:
                     COMPOSER_TOKEN: ${{ secrets.ACCESS_TOKEN }}
 
             # install only prod dependencies - do not use ramsey, it uses cache including "dev", we want to avoid it here
-            -   run: composer install --no-dev --ansi
+            -   run: |
+                    composer install --no-dev --ansi
+                    composer remove phpstan/phpstan
+                    composer clear-cache
+                    composer config repositories.phpstan/phpstan-src vcs https://github.com/phpstan/phpstan-src
```

Comment:
@TomasVotruba I updated to use `config config repositories` command to register `phpstan/phpstan-src` repo